.github/renovate.json

```diff
@@ -69,7 +69,7 @@
     {
       "matchPackageNames": ["com_google_googletest", "google/googletest"],
       "groupName": "GoogleTest",
-      "versioning": "loose"
+      "versioning": "regex:^release-(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)?$"
```

Comment:
If the patch is optional, should its separating period be inside the group?

Reply:
Yeah, that is strange. I copied the example from here without thinking about it: https://docs.renovatebot.com/modules/versioning/#using-a-custom-regex-versioning-scheme

I'll send a follow up to just make the patch mandatory.